components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/SubscriptionServices.cs

```diff
@@ -1287,6 +1330,67 @@ private class MonitoredItemWrapper {
                 return eventFilter;
             }
 
+            public void ProcessMonitoredItemNotification(SubscriptionNotificationModel message, MonitoredItemNotificationModel monitoredItemNotification) {
+                var pendingAlarmsOptions = EventTemplate?.PendingAlarms;
+                if (pendingAlarmsOptions?.IsEnabled == true && monitoredItemNotification.Value.GetValue(typeof(EncodeableDictionary)) is EncodeableDictionary values) {
+                    var conditionIdIndex = pendingAlarmsOptions.ConditionIdIndex ?? -1;
```

Comment:
Why make it -1 when it's already a type that you can check with HasValue?

Reply:
Yeah, I must have been onfocused when I wrote that code. Fixed